packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -380,17 +459,83 @@ export class Stepper implements LocalizedComponent, T9nComponent {
     });
   }
 
-  private handleDefaultSlotChange = (event: Event): void => {
-    const slottedItems = slotChangeGetAssignedElements(event);
-    const items = slottedItems.filter((el) => el?.tagName === "CALCITE-STEPPER-ITEM");
-    const spacing = Array(items.length).fill("1fr").join(" ");
-    this.el.style.gridTemplateAreas = spacing;
-    this.el.style.gridTemplateColumns = spacing;
-    this.setStepperItemNumberingSystem();
-  };
+  private renderAction(position: Position): VNode {
+    const isPositionStart = position === "start";
+    const path = isPositionStart ? "chevron-left" : "chevron-right";
+    const { currentPosition, singleViewMode, layout } = this;
+    const dir = getElementDir(this.el);
+    const totalItems = this.items.length;
+    const id = `${this.guid}-${isPositionStart ? "start" : "end"}`;
+
+    return layout === "horizontal" && singleViewMode ? (
+      <calcite-action
+        alignment="center"
+        appearance="transparent"
+        class={{
+          [CSS.actionIcon]: true,
+        }}
+        compact={true}
+        disabled={
+          (currentPosition === 0 && isPositionStart) ||
+          (currentPosition === totalItems - 1 && !isPositionStart)
+        }
+        icon={path}
+        iconFlipRtl={dir === "rtl"}
+        id={id}
+        // eslint-disable-next-line react/jsx-no-bind
+        onClick={(event) => this.handleActionClick(event, position)}
+        scale={this.scale}
+        text={isPositionStart ? "Previous Step" : "Next Step"}
+      />
+    ) : null;
+  }
+
+  private handleActionClick(event: MouseEvent, position: Position): void {
+    event.stopPropagation();
```

Comment:
Is it possible to remove this? We should not be stopping clicks from propagating.

Reply:
I am trying hard to recall, why this was added in first place. Will remove it and see if any test fails.